addon/webextension/background/startBackground.js

```diff
@@ -69,6 +71,30 @@ this.startBackground = (function() {
     return true;
   });
 
+  // We delay this check (by CHECK_MIGRATION_DELAY) just to avoid piling too
+  // many things onto browser/add-on startup
+  setTimeout(() => {
```

Comment:
Instead of just blindly setting a timeout, let's wait for the browser to become idle, via `requestIdleCallback`